### PR TITLE
i18n for some of our list views

### PR DESCRIPTION
**Pull request by mazameli**
EntitySearch.jsx
EntityList.jsx
Dashboards.jsx
DashboardsArchive.jsx

### CODE REVIEW

File: frontend/src/metabase/questions/containers/EntityList.jsx [hunk 1]
```diff
@@ -57,45 +58,45 @@ const mapDispatchToProps = {
...
         icon: 'archive',
-        empty: 'If you no longer need a question, you can archive it.'
+        empty: t`If you no longer need a question, you can archive it.`
     }
 ];
 
 const DEFAULT_SECTION = {
     icon: 'all',
-    empty: 'There aren\'t any questions matching that criteria.'
+    empty: t`There aren\'t any questions matching that criteria.`
```
Comment (kdoh) [inline frontend/src/metabase/questions/containers/EntityList.jsx]: You don't need this back slash anymore now that this is converted to use the template literal.

**Review by kdoh — CHANGES_REQUESTED** [review]
There are a couple strings in EntityList that need to have `\` removed now that they're converted but after that this looks good.

File: frontend/src/metabase/questions/containers/EntityList.jsx (hunk 1 above)
Comment (mazameli) [reply frontend/src/metabase/questions/containers/EntityList.jsx]: Gotcha, that makes sense.

**Comment by mazameli** [comment]
I removed the backslashes, so I'll go ahead and merge this one if there wasn't anything else you saw.

**Review by kdoh — APPROVED** [review]

**Pull request merged**